test/auth.js

```diff
@@ -0,0 +1,105 @@
+import chaiHttp from 'chai-http';
```

Comment:
I believe this file was renamed to `auth.test.js`. Could check if you don't already have it ? if you do then you can delete this file.